projects/plugins/super-cache/wp-cache.php

```diff
@@ -392,6 +392,9 @@ function wpsc_ajax_activate_boost() {
  * Show a Jetpack Boost installation banner (unless dismissed or installed)
  */
 function wpsc_jetpack_boost_install_banner() {
+	if ( ! wpsc_is_boost_current() ) {
```

Comment:
Though this isn't part of this PR, Boost versions have moved on since the function was defined and it's still comparing with `3.4.0`.

Reply:
bumped to 3.4.5